packages/contracts/contracts/tokens/rDAI.sol

```diff
@@ -7,18 +7,47 @@ pragma solidity >=0.5.10 <0.6.0;
 pragma experimental ABIEncoderV2;
 
 import { RToken, IAllocationStrategy } from "../RToken.sol";
+import { IERC20 } from "../IRToken.sol";
 
 /**
  * @notice RToken instantiation for rSAI (Redeemable SAI)
  */
 contract rDAI is RToken {
 
+	address private _tokenManager;
+	address private constant DAI = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+	address private constant cDAI = 0x5d3a536E4D6DbD6114cc1Ead35777bAB948E3643;
+	
+	function tokenManager() public view returns (address) {
+		return _tokenManager;
+	}
+	
+	modifier onlyTokenManager() {
+		require(msg.sender == _tokenManager, "Only token manager can call this function");
+		_;
+	}
+	
+	function setTokenManager(address newManager) onlyTokenManager external {
+		require(
+				newManager != address(0),
+				"New token manager is the zero address"
+			);
+		_tokenManager = newManager;
+	}
+	
+	function sweepTokens(IERC20 token) external {
```

Comment:
@hellwolf Do you have any objections for this being here? Personally I think it makes sense, since this functionality is specific to this particular deployment of the rToken